esbuild/esbuild.config.js

```diff
@@ -62,7 +62,7 @@ const main = () => {
     let serverProcess = null
     chokidar.watch(['dist']).on('all', () => {
       if (serverProcess) serverProcess.kill()
-      serverProcess = spawn('node', ['-r', 'dotenv/config', 'dist/server.js'], { stdio: 'inherit' })
+      serverProcess = spawn('node', ['--env-file=.env', 'dist/server.js'], { stdio: 'inherit' })
```

Comment:
Won't this be different for int tests? Not sure how it all relates to esbuild.

Reply:
yes, it will - added a separate flag for the test server..